core/gather/driver/wait-for-condition.js

```diff
@@ -498,9 +498,8 @@ async function waitForFullyLoaded(session, networkMonitor, options) {
       if (await isPageHung(session)) {
         log.warn('waitFor', 'Page appears to be hung, killing JavaScript...');
         // We don't await these, as we want to exit with PAGE_HUNG
-        void session.sendCommand('Emulation.setScriptExecutionDisabled', {value: true})
-          .catch(_ => {});
-        void session.sendCommand('Runtime.terminateExecution').catch(_ => {});
+        void session.sendCommandAndIgnore('Emulation.setScriptExecutionDisabled', {value: true});
+        void session.sendCommandAndIgnore('Runtime.terminateExecution');
```

Comment:
hmmm yeah. i'm up for tweaking it.

the `void *` thing (generally a convention from the devtools codebase) is for saying.. i'm not awaiting or returning this promise.  https://typescript-eslint.io/rules/no-floating-promises/#ignorevoid

but the new AndIgnore method is for.. catching the rejection so it doesn't continue (or go unhandled)

they're slightly different things, right?

and it seems like we have cases where we want to await the AndIgnore calls .. and cases where we don't want to await them
So i think this new method should still return a promise. but.. it can be empty

-------

and given this tweak, maybe  `ignore` isn't a great fit. wdyt `sendCommandQuietly`?  open to other names.  (but also no big whoop on naming)